src/main/java/frc/robot/RobotContainer.java

```diff
@@ -91,26 +98,31 @@ public RobotContainer()
                 _notepath = new Notepath(new NotepathIOSparkMax());
                 _shooterBed = new ShooterBed(new ShooterBedIOVictorSPX());
                 _shooterFlywheel = new ShooterFlywheel(new ShooterFlywheelIOSparkMax());
+                _climb = new Climb(new ClimbIOVictorSPX());
```

Comment:
The `Climb` subsystem also needs a reference to the gyro:

```suggestion
                _climb = new Climb(_gyro, new ClimbIOVictorSPX());
```